### DISCUSSION TITLE
Using this inside a grpc service

### DISCUSSION THREAD

**Discussion by dvaldivia** [kamalshkeir/kago#3] [General]
Hi! I love what this project brings to the table, I use to develop a lot in Django and miss it greatly since I started doing golang, I'm curious if I were to try to run this inside a grpc service, mainly to leverage the ORM features, what would be needed?

**Comment by kamalshkeir** [comment]
Hello, thank you for your support
Django is great, i love this framework, and really that's why i decided to make a framework similar to Django, with the ease of use and the convenience behind the admin, the shell, auto migrations ..., an example of these hidden gems from django : i actually put the request and the user in every Html context to be able to directly check if user is authenticated or request path,... 

Answering your question: you need nothing, i even tested all kind of grpc request with python client, and golang client, i can get you the example i have done if find yourself blocked, but it should be very straightforward
# FOR GOLANG
1- go get google.golang.org/grpc

2- make folder generated

3- make users.proto with message and service and add `option go_package = "./;generated";` after `syntax = "proto3";`

4- protoc --go_out=plugins=grpc:gRPC/generated .\gRPC\protos\database.proto

# FOR PYTHON
1- pip install grpcio-tools

2- python -m grpc_tools.protoc -I./protos --python_out=./generated/ --grpc_python_out=./generated/ ./protos/database.proto
